ci/check-code-gen.sh

```diff
@@ -9,11 +9,8 @@ if [ ! -f .gitignore ]; then
 fi
 
 
-git init
-git config user.email "you@example.com"
+git config --global user.email "you@example.com"
```

Comment:
changing global git config is a bit scary here.
i think we can remove this as we are not doing a git commit.
if we are doing a git commit, can we add the author using the `--author "Your Name <you@example.com>"` flag in the git commit command?

Reply:
deleted as we're no longer doing a commit in this step